CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+## v0.73.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.73.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.73.0) and the [opentelemetry-collector-contrib v0.73.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.73.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+- [Add java zc options to linux installer script](https://github.com/signalfx/splunk-otel-collector/pull/2718)
+- [Update installer script for generate_service_name option](https://github.com/signalfx/splunk-otel-collector/pull/2730)
```

Comment:
```suggestion
- [Added profiling and JVM metrics options to linux installer script for zero configuration auto instrumentation of Java apps](https://github.com/signalfx/splunk-otel-collector/pull/2718)
- [Update installer script with option to generate service name with zero configuration auto instrumentation of Java apps](https://github.com/signalfx/splunk-otel-collector/pull/2730)
```

Reply:
I think we can remove the reference to #2730 since it was just to address an issue from #2718.

Reply:
Makes sense, we should update this to reflect this option as well then (Added profiling, JVM metrics and service name generation options for zero configuration auto instrumentation of Java apps).